scrapy_poet/page_input_providers.py

```diff
@@ -371,7 +374,7 @@ async def __call__(
                 )
 
             page_object = po_instances[page_object_cls]
-            item = await page_object.to_item()
+            item = yield from ensureDeferred(page_object.to_item())
```

Comment:
Does it work with page objects which use asyncio?

Reply:
Addressed in eaf0d40